mod/badges/spec/support/award_answer_create_badges_shared_examples.rb

```diff
@@ -15,21 +15,21 @@
   end
 
   describe "badges earned pointer" do
-    it "exists" do
-      expect(badge_pointer).to be_instance_of Card
-    end
+    # it "exists" do
+    #   expect(badge_pointer).to be_instance_of Card
+    # end
```

Comment:
what changed here?

Reply:
I reduced the number of tests while I was debugging this (since these are shared examples it's not possible to run them one by one)